src/main/java/org/openhab/binding/zigbee/converter/ZigBeeConverterColorColor.java

```diff
@@ -106,72 +266,186 @@ public void run() {
                     return;
                 }
 
-                int level;
                 try {
                     if (command instanceof HSBType) {
-                        HSBType hsb = (HSBType) command;
-                        clusterColorControl
-                                .moveToHueAndSaturationCommand((int) (hsb.getHue().doubleValue() * 254.0 / 360.0 + 0.5),
-                                        (int) (hsb.getSaturation().doubleValue() * 254.0 / 100.0 + 0.5), 10)
-                                .get();
-                        level = hsb.getBrightness().intValue();
+                        HSBType color = (HSBType) command;
+                        PercentType brightness = color.getBrightness();
+
+                        if (brightness.intValue() != currentHSB.getBrightness().intValue()) {
+                            changeBrightness(brightness);
+                            // Wait for transition to complete
+                            // Some lights do not like receiving a level/color change command
+                            // while the previous transition is in progress...
+                            Thread.sleep(1000);
+                        }
+
+                        if (supportsHue) {
```

Comment:
As mentioned earlier, I don't know if there's a need for both "supports" flags and we have an ambiguous case here - what if both are false? I think one flag is better - or an enum to define the mode?